packages/frontend/vite.config.js

```diff
@@ -59,6 +59,7 @@ export default defineConfig({
     },
   },
   build: {
+    minify: true,
```

Comment:
it would need to be introduced outside of the scope of this PR

Reply:
I am against removing the minify